docs/docs/working-with-images.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Working With Images In Gatsby
+title: Working With Images in Gatsby
```

Comment:
There is still a uppercase "with" here. 🙈

Reply:
eagle eyes 👀😂
Thanks for spotting - fixed it now!